### PR TITLE
Accepted papers

### PR DESCRIPTION
**Pull request by maximilianigl**
Please try on a local version whether the videos and download links work.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -178,6 +178,10 @@ Below times are in Detroit time. Current time in Detroit is <span id="centraltim
 
 {% include schedule %}
 
+## Papers
```
Comment (m-naumann) [inline index.md]: Is this intentionally above the invited speaker list?

**Review by m-naumann — DISMISSED** [review]
Looks great, thanks! Do we want to sneak this or only share it on the workshop day?

**Comment by maximilianigl** [comment]
I've put it behind the speakers now.
I'm fine having the papers already there - or is there a reason we should wait?

**Review by m-naumann — APPROVED** [review]

**Comment by m-naumann** [comment]
> I've put it behind the speakers now.
> I'm fine having the papers already there - or is there a reason we should wait?

No all good, just approved

**Pull request merged**